README.md

```diff
@@ -167,6 +169,7 @@ Interested in contributing? If so, click [here](https://github.com/KinsonDigital
 
 [![twitter-logo](https://raw.githubusercontent.com/KinsonDigital/.github/master/Images/twitter-logo-16x16.svg)Calvin Wilkinson](https://twitter.com/KDCoder) (KinsonDigital GitHub Organization - Owner)
 
+
```

Comment:
Remove empty line and add 2 spaces to the end of the first twitter markdown item